gradle.properties

```diff
@@ -1,2 +1,3 @@
 org.gradle.caching=true
 org.gradle.parallel=true
+org.gradle.jvmargs='-Dfile.encoding=UTF-8'
```

Comment:
any specific reason for this?

Reply:
Unit tests are failing in windows OS due to an encoding issue. @hasithaa can verify if this solves the issue?

Reply:
Yes, this solves test failures in Windows. Still, we need https://github.com/wso2/code-quality-tools/pull/9 to build on windows.